### PR TITLE
Reference page to download windows installer

### LINKED ISSUE #1366: Docs feedback: /getting-started/install-agent-on-windows.md
This doc does not currently contain a link to the Windows Agent from https://github.com/grafana/agent/releases.

### PR DESCRIPTION
**Pull request by rfratto**
Fixes #1366

### CODE REVIEW

File: docs/getting-started/install-agent-on-windows.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ weight = 120
 
 The installer will install Grafana Agent into the default directory `C:\Program Files\Grafana Agent`. The [windows_exporter integration](https://github.com/prometheus-community/windows_exporter) can be enabled with all default windows_exporter options.
 
+The installer can be retrieved for the latest [Grafana Agent release](https://github.com/grafana/agent/releases/latest) by downloading the release asset named `grafana-agent-installer.exe`.
```
Comment (mattdurham) [inline docs/getting-started/install-agent-on-windows.md:10]: `The installer can be retrieved for the latest [Grafana Agent release](https://github.com/grafana/agent/releases/latest) by downloading [grafana-agent-installer.exe](https://github.com/grafana/agent/releases/latest/download/grafana-agent-installer.exe).`

Directly link?

Comment (rfratto) [reply docs/getting-started/install-agent-on-windows.md:10]: Hmm, I don't know, it feels like we're asking people to trust random links that download executables.

Comment (mattdurham) [reply docs/getting-started/install-agent-on-windows.md:10]: We already do this on the Releases page for better or worse.

Comment (rfratto) [reply docs/getting-started/install-agent-on-windows.md:10]: WDYT about doing both to help people who are hesitant?

**Review by mattdurham — APPROVED** [review]
LGTM

**Pull request merged**